pipelines/treatment/bilhetagem_processos_manuais/tasks.py

```
@@ -67,11 +67,19 @@ def get_gaps_from_result_table(
         """
+
     df_bq = pandas_gbq.read_gbq(query, project_id=project_id)
     for table_id in table_ids:
         df = df_bq[df_bq["table_id"] == table_id]
+        timestamps = df["timestamp_captura"].to_list()
         result[table_id] = {
-            "timestamps": df["timestamp_captura"].to_list(),
+            "timestamps": timestamps,
+            "recapture_params": [
+                [
+                    {"recapture": True, "recapture_timestamps": timestamps[i : i + 20]}  # noqa
```

Ruff (0.13.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)